README.md

```diff
@@ -109,7 +109,7 @@ The expected changes are excluded before generating digests for data in the `bui
 
 An additional script has been created to generate a report that enables comparing the totals for candidates.  The script is `bin/report-candidates.py` and it generates `build/candidates.csv` and `build/candidates.xlsx`. The reports include a list of all the candidates and totals calculated multiple ways that should add up to the same number.
 
-To ensure that database schema changes are visible in pull requests, the complete postgres schema is also save to a `schema.sql` file in the `build` directory.  Because the `build` is automatically re-built for each branch in a PR and committed, any change to the schema caused by a code change will be shown a difference in the `schema.sql` file when reviewing the PR.
+To ensure that database schema changes are visible in pull requests, the complete postgres schema is also saved to a `schema.sql` file in the `build` directory.  Because the `build` directory is automatically re-built for each branch in a PR and committed to the repository, any change to the schema caused by a code change will be shown a difference in the `schema.sql` file when reviewing the PR.
```

Comment:
I don’t understand how a code change would change the schema.  The schema is generated from dbschema. Those changes are committed to the repository. It seems schema.sql is redundant.

Reply:
Keep in mind that dbschema is an input and schema.sql is an output used to verify that the input was correctly processed to generate the output that we are expecting.  You're assuming that the schema setup code is working correctly indefinitely.  If the code that generates the schema from dbschema is 100% guaranteed to create the schema that you're expecting until the end of time, no matter what version of csvkit you upgrade to or whatever changes you may have done to re-organize the code or change the code in any way, then there would not be a need for schema.sql.   The fact of the matter is that mistakes are made in the code and the libraries that we use.  We're not going to catch those mistakes if we don't compare new results against old results.  This PR is the very first use of schema.sql.  It lets us know how upgrading csvkit, switching over to dbschema and importing new data has changed the schema compared to the master branch.  Without schema.sql, there's no way right now to know what schema is being used in the master branch and as a result, no way of knowing whether we've completely changed the schema by upgrading csvkit.